.codecov.yml

```diff
@@ -1,6 +1,3 @@
-codecov:
```

Comment:
So you don't need a token if it's public?

Reply:
The token had changed which is what broke things, but anymore you don't need a token if it's running in travis.